hack/make-release-notes.sh

```diff
@@ -34,6 +38,22 @@ download_assets=(
   krew.yaml
 )
 
+install_release_notes() {
+  relnotes_dir="$(mktemp -d)"
+  trap 'rm -rf -- ${relnotes_dir}' EXIT
+
+  cd "$relnotes_dir"
+  go mod init foo
+  GOBIN="${SCRIPTDIR}" go install github.com/corneliusweig/release-notes
```

Comment:
I recall the whole point of this mktemp trick was to pin it to a particular version :)